### PR TITLE
Changelog for recent changes

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ of tasks being currently executed.
   - Removed unnecessary fields from mailbox mapping
   - Disable dynamic mapping thanks to a change of the header structure 
   - Read related [upgrade instructions](upgrade-instructions.md)
+  
+### Fixed
+- JAMES-2828 & JAMES-2928 bugs affecting JDBCMailRepository usage with PostgresSQL
```
Comment (Arsnael) [inline CHANGELOG.md]: Was thinking... when it's a fix done by a contributor not part of our team, shouldn't we mention his name with a little thanks in the changelog?

Comment (trantienduchn) [reply CHANGELOG.md]: I don't see how the [JAMES-2928](https://issues.apache.org/jira/browse/JAMES-2928) relates to JDBCMailRepository

File: upgrade-instructions.md [hunk 2]
```diff
@@ -27,13 +27,20 @@ SHA-1 0d72783ff4
...
 
 Concerned product: Guice product relying on ElasticSearch
 
 We significantly improved our usage of ElasticSearch. Underlying changes include:
 
  - The use of routing to collocate emails of a same mailbox within a same shard. This enables search queries to avoid cluster
  level synchronisation, and thus enhance throughput, latencies and scalability.
+ - Disabling dynamic mapping. We now represent headers as nested objects.
+ - Removing some not needed fields from the mapping
+ - No longer index raw HTML. This was possible under some configuration combinaison, and caused the data stored in elasticSearch
```
Comment (trantienduchn) [inline upgrade-instructions.md]: s/combinaison/combination

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,10 +5,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Added
+- Distributed task management for Guice cassandra-rabbitmq product. This enables two James server to share a consistent view
```
Comment (trantienduchn) [inline CHANGELOG.md]: s/server/servers

File: CHANGELOG.md (hunk 1 above)
Comment (Arsnael) [reply CHANGELOG.md]: Good catch, it's JAMES-2929 instead

**Comment by chibenwa** [comment]
> I don't see how the JAMES-2928 relates to JDBCMailRepository

That why I wrote JAMES-2828

**Comment by Arsnael** [comment]
I still didn't get an answer to my first question though... Should we mention external contributors on the changelog when a fix is from them? I think it's good to be honest, I think it can give the feeling from us that we really appreciate people contributing to the project and help strengthen the community

**Comment by chibenwa** [comment]
> Should we mention external contributors on the changelog when a fix is from them? 

You are right, we should.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,10 +5,35 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Added
+- Distributed task management for Guice cassandra-rabbitmq product. This enables two James servers to share a consistent view
```
Comment (remk) [inline CHANGELOG.md]: Why only two ?

**Comment by chibenwa** [comment]
:green_apple: before CI restart

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**